news/_posts/releases/2022-01-26-prestashop-1-7-8-3-maintenance-release.md

```diff
@@ -0,0 +1,82 @@
+- --
+layout: post
+title:  "Release Of PrestaShop 1.7.8.3"
+subtitle: "MAINTENANCE VERSION OF THE 1.7.8.X BRANCH"
+date:   2022-01-26 14:00:00
+authors: [ PrestaShop ]
+image: /assets/images/2022/01/1.7.8.3_banner.jpg
+icon: icon-checked
+tags:
+- version
+- patch
+- releases
+
+---
+
+PrestaShop 1.7.8.3 is now available. This maintenance release fixes 25 bugs and 1 security issue since the 1.7.8.2 release.
+Note : two of these fixes are not fully backward compatible ( [#26933](https://github.com/PrestaShop/PrestaShop/issues/26933) and [#26643](https://github.com/PrestaShop/PrestaShop/issues/26643) ) 
+
+*![1.7.8.3 is available!](/assets/images/2022/01/1.7.8.3_banner.jpg)*
+
+
+We suggest upgrading your shop quickly in order to benefit from these fixes. Of course, don’t forget to backup before.
```

Comment:
```suggestion
We suggest upgrading your store quickly to benefit from these corrections. Make sure to perform a full manual backup before. It will be essential to roll back to the previous database and files if needed.
```

Reply:
```suggestion
We suggest upgrading your store quickly to benefit from these improvements. Make sure to perform a full manual backup before, so you can roll back the upgrade if something goes wrong.
```